components/ArtPieces/index.js

```diff
@@ -0,0 +1,22 @@
+import Card from "../CardComponent";
+
+export default function ArtPiecesPage({ data }) {
+  return (
+    <>
+      <h1>Pieces</h1>
+      <div>
```

Comment:
Since you are rendering `<li>` items maybe wrapping them with a `<ul>` would be better for semantic reasons 😊